### PR TITLE
new blog post: Announcing Plugins and Cloud Security with Falco

### PR DESCRIPTION
**Pull request by ldegio**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area blog

> /area documentation

> /area videos

**What this PR does / why we need it**:

This blog post talks about the newly released plugin infrastructure and how Falco is expanding into the domain of cloud security.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/blog/announcing_plugins_and_cloud_security_with_falco.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Plugins extend the applicability of this philosophy to an endless number of new domains. One of these domains is cloud security.
+
+### Runtime Security: a Better Option for Threat Detection in the Cloud
+Cloud security is a fertile and constantly evolving space. When implementing cloud security you can choose among many different options. However, architecturally, most options fall in one of the following categories:
+
+1. Tools that query the cloud APIs or watch cloud data stores to detect misconfigurations or vulnerabilities
+2. Tools that stream cloud logs into a backend, index them and let you query them
+
+To detect threats in cloud-based software, Category 1 is not very useful. Polling is great to detect gaps and validate compliance, but lacks the real time nature required to detect threats and respond quickly. Category 2 is powerful, but it’s also tremendously expensive (especially in environments like the public cloud where tons of logs are produced) and not friendly to deploy and use.
```
Comment (leogr) [inline content/en/blog/announcing_plugins_and_cloud_security_with_falco.md:48]:
```suggestion
To detect threats in cloud-based software, Category 1 is not very useful. Polling is great to detect gaps and validate compliance, but lacks the real-time nature required to detect threats and respond quickly. Category 2 is powerful, but it’s also tremendously expensive (especially in environments like the public cloud where tons of logs are produced) and not friendly to deploy and use.
```

File: content/en/blog/announcing_plugins_and_cloud_security_with_falco.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+I argue that the Falco runtime security approach is the ideal one. Falco consumes few resources and, most importantly, it analyzes the data in a streaming way. No need to perform expensive copies, no need to wait until the data is indexed. Falco looks at your data in real time and notifies you in seconds.
```
Comment (leogr) [inline content/en/blog/announcing_plugins_and_cloud_security_with_falco.md:50]:
```suggestion
I argue that the Falco runtime security approach is the ideal one. Falco consumes few resources and, most importantly, it analyzes the data in a streaming way. No need to perform expensive copies, no need to wait until the data is indexed. Falco looks at your data in real-time and notifies you in seconds.
```

File: content/en/blog/announcing_plugins_and_cloud_security_with_falco.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Getting up and running with Falco takes only a few minutes, and adopting it for both cloud logs and system calls allows a unified approach to threat detection.
+
+### What is the future for Falco?
+
+V0.31.0 comes with one plugin, Cloudtrail, but expect many more to come in the future. Our vision is making Falco the runtime policy engine for, well, everything. :-) We want to support all the clouds, and include more services from each of them.
```
Comment (leogr) [inline content/en/blog/announcing_plugins_and_cloud_security_with_falco.md:56]:
```suggestion
V0.31.0 comes with one plugin, Cloudtrail, but expect many more to come in the future. Our vision is to make Falco the runtime policy engine for, well, everything. :-) We want to support all the clouds, and include more services from each of them. 
```

File: content/en/blog/announcing_plugins_and_cloud_security_with_falco.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+
+You can find us in the [Falco community](https://github.com/falcosecurity/community). Please feel free to reach out to us for any questions, suggestions, or even for a friendly chat!
+
+If you would like to find out more about Falco:
+
+* Get started in [Falco.org](http://falco.org/)
+* Check out the [Falco project in GitHub](https://github.com/falcosecurity/falco).
+* Get involved in the [Falco community](https://falco.org/community/).
+* Meet the maintainers on the [Falco Slack](https://kubernetes.slack.com/?redir=%2Farchives%2FCMWH3EH32).
```
Comment (leogr) [inline content/en/blog/announcing_plugins_and_cloud_security_with_falco.md:68]:
```suggestion
* Meet the maintainers on the [Falco Slack](https://kubernetes.slack.com/messages/falco).
```

**Review by leogr — COMMENTED** [review]
Awesome blog, Loris! :star_struck: 

Just found a couple of minor issues (mostly nitpicking :smile_cat: )

**Comment by jasondellaluce** [comment]
Love it! I think this is great.

**Review by danpopnyc — APPROVED** [review]
great work Loris! /lgtm

**Review by danpopnyc — APPROVED** [review]

**Pull request merged**